src/internal/watch/postgres/impl.go

```diff
@@ -0,0 +1,144 @@
+package postgres
+
+import (
+	"strconv"
+	"strings"
+	"sync"
+
+	"github.com/lib/pq"
+	"github.com/pachyderm/pachyderm/v2/src/internal/collection"
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/pachsql"
+)
+
+type EventType int
```

Comment:
If this goes into the database it shouldn't be a platform dependent type.  Pick an integer type.

Reply:
This can be rebased after the other branch goes in.